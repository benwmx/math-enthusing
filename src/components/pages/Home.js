import React from 'react';

const Home = () => (
  <div className="home-container">
    <h2 className="title">Welcome to our Page</h2>
    <p className="home-details">
      lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incidid
      lorem inc lorem ipsum dolor sit amet, consectetur adiplore, sed do eius mod
      lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incididunt
      lorem inc lore sit amet et alice. Lore m lorem ips lorem  ipsum dolor sit amet,  consectet
      lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor
    </p>
  </div>
);

export default Home;
