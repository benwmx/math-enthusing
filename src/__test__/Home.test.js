import React from 'react';
import renderer from 'react-test-renderer';
import Home from '../components/pages/Home';

describe('Home component test', () => {
  it('Should render', () => {
    const home = renderer.create(<Home />).toJSON();
    expect(home).toMatchSnapshot();
  });
});
