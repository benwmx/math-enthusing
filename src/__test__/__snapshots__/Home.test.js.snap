// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home component test Should render 1`] = `
<div
  className="home container"
>
  <h2
    className="title"
  >
    Welcome to our Page
  </h2>
  <p
    className="home-details"
  >
    lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incidid lorem inc lorem ipsum dolor sit amet, consectetur adiplore, sed do eius mod lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incididunt lorem inc lore sit amet et alice. Lore m lorem ips lorem  ipsum dolor sit amet,  consectet lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incidid lorem inc lorem ipsum dolor sit amet, consectetur adiplore, sed do eius mod lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incididunt lorem inc lore sit amet et alice. Lore m lorem ips lorem  ipsum dolor sit amet,  consectet lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incidid lorem inc lorem ipsum dolor sit amet, consectetur adiplore, sed do eius mod lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor incididunt lorem inc lore sit amet et alice. Lore m lorem ips lorem  ipsum dolor sit amet,  consectet lorem ipsum dolor sit amet, consectetur adiplore, sed do eiusmod tempor
  </p>
</div>
`;
